app/javascript/app/components/country/laws-and-policies/laws-and-policies-selectors.js

```diff
@@ -0,0 +1,141 @@
+import { createSelector, createStructuredSelector } from 'reselect';
+import groupBy from 'lodash/groupBy';
+import isEmpty from 'lodash/isEmpty';
+
+const CARDS_IN_ROW = 2;
+
+const getIso = state => state.iso || null;
+
+const getData = state => state.lawsAndPolicies || null;
+
+const getActiveSector = state => state.activeSector;
+
+export const getSectors = createSelector([getIso, getData], (iso, { data }) => {
+  const sectors = (data && data[iso] && data[iso].sectors) || [];
```

Comment:
An empty array won't cache the null result. We can try to use null instead if its possible

Reply:
👍 although the empty array has its advantages as you don't have to do additional checks for map, find, etc. functions